package.json

```diff
@@ -5,18 +5,24 @@
   "license": "MIT",
   "author": "Bjorn Lu",
   "bin": {
-    "create-vite-extra": "index.js"
+    "create-vite-extra": "./dist/index.js"
+  },
+  "scripts": {
+    "start": "node dist/index.js",
+    "build": "tsc index.ts",
+    "format": "prettier --write .",
+    "update-deps": "pnpm update -r --latest && node update-deno-deps.js"
   },
```

Comment:
Can the scripts be moved to the spot like before? Also the `build` script doesn't seem to use tsup